lib/success.js

```diff
@@ -106,7 +106,7 @@ module.exports = async (pluginConfig, context) => {
                   type: 'mrkdwn',
                   text: `:package: *<${repo.URL}|${repo.path}>:*   <${
                     repo.URL
-                  }/releases/tag/${gitTag}|${gitTag}>`
+                  }/releases/${gitTag}|${gitTag}>`
```

Comment:
Does this work with GitHub and Bitbucket?

Reply:
Yes you're totally right! I've just made an update to manage **Gitlab** and **Github** tag url in your default template.
Nothing changes for Github projects, the tag url is still: `${repo.URL}/releases/tag/${gitTag}` and now for Gitlab projects, the url is: `${repo.URL}/-/releases/${gitTag}` ;)

I don't know what's the url for Bitbucket projects but if needed feel free to add a new condition later.